examples/panda/panda_goal_reaching/controllers/robot_supervisor_manager/DDPG_runner.py

```diff
@@ -0,0 +1,85 @@
+from numpy import convolve, ones, mean, random
+
+from robot_supervisor_ddpg import PandaRobotSupervisor
+from agent.ddpg import DDPGAgent
+
+
+def run():
+    # Initialize supervisor object
+    env = PandaRobotSupervisor()
+
+    # The agent used here is trained with the DDPG algorithm (https://arxiv.org/abs/1509.02971).
+    # We pass (10, ) as numberOfInputs and (7, ) as numberOfOutputs, taken from the gym spaces
+    agent = DDPGAgent(alpha=0.000025, beta=0.00025, input_dims=[env.observation_space.shape[0]], tau=0.001, batch_size=64,  layer1_size=400, layer2_size=400, n_actions=env.action_space.shape[0]) 
+              
+    # agent.load_models() # Load the pretrained model
+    episodeCount = 0 
+    episodeLimit = 50000
+    solved = False  # Whether the solved requirement is met
+
+    # Run outer loop until the episodes limit is reached or the task is solved
+    while not solved and episodeCount < episodeLimit:
+        state = env.reset()  # Reset robot and get starting observation
+        env.episodeScore = 0
+
+        print("===episodeCount:", episodeCount,"===")
+        env.target = env.getFromDef("TARGET%s"%(random.randint(1, 10, 1)[0])) # Select one of the targets
+        # Inner loop is the episode loop
+        for step in range(env.stepsPerEpisode):
+            # In training mode the agent returns the action plus OU noise for exploration
+            act = agent.choose_action(state)
+
+            # Step the supervisor to get the current selectedAction reward, the new state and whether we reached the
+            # the done condition
+            newState, reward, done, info = env.step(act*0.032)
```

Comment:
Just a question: why you multiply action by 0.032?

Reply:
Change all actions from [-1,1] to [-0.032, 0.032] since I think that taking "1"or"-1" for actions is too large.